server/athenian/api/models/web/release_metric_id.py

```diff
@@ -0,0 +1,7 @@
+from athenian.api.models.web.base_model_ import Enum, Model
+
+
+class ReleaseMetricID(Model, metaclass=Enum):
```

Comment:
FYI, the next release metrics are going to be:

release-size (in terms of PRs, commits and loc)
release-frequency (avg release age)
The designs and Jira tickets associated are still in progress but in case you want to tackle all of them at once, I can adapt the priorities.

Reply:
OK, I'll add all of them to the spec but will not code yet.